src/libtorch.cc

```diff
@@ -810,10 +844,27 @@ ModelInstanceState::Execute(
   try {
     // enable/disable optimized execution
     torch::jit::setGraphExecutorOptimize(
-        !model_state_->DisabledOptimizedExecution());
+        model_state_->EnabledOptimizedExecution());
 
     // enable/disable inference mode - supersedes NoGradGuard
-    torch::InferenceMode infer_guard(model_state_->InferenceMode());
+    torch::InferenceMode infer_guard(model_state_->EnabledInferenceMode());
+
+    // NvFuser is only supported for GPU instances. No change is made if
+    // enable_nvfuser is not set by user
```

Comment:
Can you add a link to how you came up with the following truth vales for different options below?

Reply:
Unfortunately, there is no user facing doc for the C++ API calls for NvFuser, I found it by digging through https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/python/init.cpp and by discussing with some pytorch framework experts.